README.md

```diff
@@ -1,2 +1,62 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Hello-Microverse
-The goal of this project is learn the concepts of version control, Github workflow, professional documentation and use of linters
+
+> The goal of this project is learn the concepts of version control, Github workflow, professional documentation and use of linters.
+
+## Built With
+
+- Major Languages : HTML and CSS
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://jonahkayizzi.github.io/Hello-Microverse/)
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@JonahKayizzi](https://github.com/JonahKayizzi)
+- Twitter: [@JonahKayizzi](https://twitter.com/JonahKayizzi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jonathan-kayizzi-bb527965)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It would be nice if you add a license file in your repository you can add using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [ ] Don't forget to add your name, current year, and app type in your MIT.md file after adding it in your project :+1:
**Note: This is the license file of your project so please take it seriously :thinking:**

Reply:
A license has also been added to the repository. Thanks for this tip 👍